recipe/build.sh

```diff
@@ -98,6 +102,11 @@ cmake -LAH                                                                \
     -DPYTHON_INCLUDE_DIR=${INC_PYTHON}                                    \
     -DPYTHON_LIBRARY=${LIB_PYTHON}                                        \
     -DOPENCV_SKIP_PYTHON_LOADER=1                                         \
+    -DZLIB_INCLUDE_DIR=${PREFIX}/include                                  \
+    -DZLIB_LIBRARY_RELEASE=${PREFIX}/lib/libz.${DYNAMIC_EXT}              \
```

Comment:
Just as an FYI, we can make this `.../libz${SHLIB_EXT}` as `conda-build` sets `${SHLIB_EXT}` to `.dylib` or `.so` as appropriate.

Reply:
mostly just copied lines from AnacondaRecipes

Reply:
No worries. Just sharing for awareness.